kphp-core/kphp-core-context.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <cstddef>
+
+#ifndef INCLUDED_FROM_KPHP_CORE
+  #error "this file must be included only from kphp_core.h"
+#endif
+
+struct CoreAllocator {
```

Comment:
What dou you think about renaming it to `RuntimeAllocator`, and try to use it everywhere in new runtime instead of `dl::allocate`?